app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,10 @@
+class GuestRoomsController < ApplicationController
+  def create
+    # pry
+    @guest = Guest.find(params[:id])
+    @room = Room.find(params[:room])
+    GuestRoom.create!(guest_id: @guest.id, room_id: @room.id)
+    
+    redirect_to "/guests/#{@guest.id}"
```

Comment:
The logic here looks good. As our apps start to get larger, you can consider abstracting logic like line 6 (creating a record in a db table) into the Model and creating a method by which the controller can call this method.